h/panels/navbar.py

```diff
@@ -29,14 +30,20 @@ def navbar(context, request, search=None, opts=None):
                 'title': group.name,
                 'link': request.route_url('group_read', pubid=group.pubid, slug=group.slug)
             })
-            groups_suggestions.append({
-                'name': group.name,
-                'pubid': group.pubid
-            })
         user_activity_url = request.route_url('activity.user_search',
                                               username=request.user.username)
         username = request.user.username
 
+    # make all groups visible in the search auto complete
+    groups = profile_groups_factory(None, request).all(request.user, request.authority)
```

Comment:
Can you pull the changes related to group suggestions into a separate PR? There are some additional issues that we'll need to resolve. For example, the autosuggest list currently displays all groups under a "Your groups" heading when you type "group:" which doesn't make sense any more if this list includes open groups.

<img width="301" alt="screenshot 2018-02-13 12 16 45" src="https://user-images.githubusercontent.com/2458/36149520-edf63c82-10b7-11e8-84f2-cc338a3f0b65.png">